tests/trash.rs

```diff
@@ -40,6 +40,20 @@ fn test_delete_file() {
     trace!("Finished test_delete_file");
 }
 
+#[test]
+#[serial]
+fn test_trash_path() {
+    init_logging();
+    trace!("Started test_trash_path");
+
+    let path = PathBuf::from(get_unique_name());
+    File::create(&path).unwrap();
+
+    delete(&path).unwrap();
```

Comment:
Maybe adjust `delete` so that it returns the path to the deleted file to make it possible to assert that it lives within the `trash_path()`.